libs/ui/elements/src/lib/kind-badge/kind-badge.component.html

```diff
@@ -1,9 +1,11 @@
-<ng-icon [name]="iconKind" class="mr-1"></ng-icon>
+<div [class]="badgeClasses" *ngIf="kind">
+  <ng-icon [name]="iconKind" class="mr-1 text-[0.85em]"></ng-icon>
 
-<ng-container *ngIf="contentTemplate; else defaultContent">
-  <ng-container *ngTemplateOutlet="contentTemplate"></ng-container>
-</ng-container>
+  <ng-container *ngIf="contentTemplate; else defaultContent">
+    <ng-container *ngTemplateOutlet="contentTemplate"></ng-container>
+  </ng-container>
+</div>
 
 <ng-template #defaultContent>
-  {{ kind | translate }}
+  {{ 'record.kind.' + kind | translate }}
```

Comment:
Maybe it would make sense to move the translation marker to this component. Currently, I can find it in two components and both with `record.kind.data` and `record.kind.dataset` keys.

Reply:
good point, also I forgot to make some cleaning after some refacto. Now the markers are in the kind badge component only.

Reply:
thanks!